packages/gcc7.rb

```diff
@@ -90,25 +89,25 @@ def self.install
 
       # Make symbolic links
       FileUtils.ln_s('gcc', "#{CREW_DEST_PREFIX}/bin/cc")
-      FileUtils.ln_s('gcc-7.4', "#{CREW_DEST_PREFIX}/bin/gcc")
-      FileUtils.ln_s('c++-7.4', "#{CREW_DEST_PREFIX}/bin/c++")
-      FileUtils.ln_s('g++-7.4', "#{CREW_DEST_PREFIX}/bin/g++")
-      FileUtils.ln_s('cpp-7.4', "#{CREW_DEST_PREFIX}/bin/cpp")
-      FileUtils.ln_s('gcc-ar-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-ar")
-      FileUtils.ln_s('gcc-nm-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-nm")
-      FileUtils.ln_s('gcc-ranlib-7.4', "#{CREW_DEST_PREFIX}/bin/gcc-ranlib")
-      FileUtils.ln_s('gcov-7.4', "#{CREW_DEST_PREFIX}/bin/gcov")
-      FileUtils.ln_s('gcov-dump-7.4', "#{CREW_DEST_PREFIX}/bin/gcov-dump")
-      FileUtils.ln_s('gcov-tool-7.4', "#{CREW_DEST_PREFIX}/bin/gcov-tool")
-      FileUtils.ln_s('gfortran-7.4', "#{CREW_DEST_PREFIX}/bin/gfortran")
+      FileUtils.ln_s("gcc-#{version}", "#{CREW_DEST_PREFIX}/bin/gcc")
+      FileUtils.ln_s("c++-#{version}", "#{CREW_DEST_PREFIX}/bin/c++")
+      FileUtils.ln_s("g++-#{version}", "#{CREW_DEST_PREFIX}/bin/g++")
+      FileUtils.ln_s("cpp-#{version}", "#{CREW_DEST_PREFIX}/bin/cpp")
+      FileUtils.ln_s("gcc-ar-#{version}", "#{CREW_DEST_PREFIX}/bin/gcc-ar")
+      FileUtils.ln_s("gcc-nm-#{version}", "#{CREW_DEST_PREFIX}/bin/gcc-nm")
+      FileUtils.ln_s("gcc-ranlib-#{version}", "#{CREW_DEST_PREFIX}/bin/gcc-ranlib")
+      FileUtils.ln_s("gcov-#{version}", "#{CREW_DEST_PREFIX}/bin/gcov")
+      FileUtils.ln_s("gcov-dump-#{version}", "#{CREW_DEST_PREFIX}/bin/gcov-dump")
+      FileUtils.ln_s("gcov-tool-#{version}", "#{CREW_DEST_PREFIX}/bin/gcov-tool")
+      FileUtils.ln_s("gfortran-#{version}", "#{CREW_DEST_PREFIX}/bin/gfortran")
 
-      FileUtils.ln_s("$(gcc -dumpmachine)-c++-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++")
-      FileUtils.ln_s("$(gcc -dumpmachine)-g++-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++")
-      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc")
-      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-ar-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar")
-      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-nm-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm")
-      FileUtils.ln_s("$(gcc -dumpmachine)-gcc-ranlib-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib")
-      FileUtils.ln_s("$(gcc -dumpmachine)-gfortran-7.4", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran")
+      FileUtils.ln_s("$(gcc -dumpmachine)-c++-#{version}", "#{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++")
```

Comment:
I'm fairly sure this will break stuff. Packages expect `gcc-8.2`, not `gcc-8.2.0`.
Note: Above is just an example, probably not relevant for this package.